WASM_Support.md

```diff
@@ -0,0 +1,28 @@
+<!--
+******************************************************************************
+* 
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/-->
+
+# WASM Support
+
+``WASM`` stands for WebAssembly, a low-level binary format for executing code in web browsers. 
+It is designed to be a portable target for compilers and to be efficient to parse and execute. 
+
+WebAssembly aims to provide a fast, efficient, and safe way to run code in web browsers without needing plugins or other software. WebAssembly code can be written in a variety of programming languages, including C, C++, Rust, and others, and then compiled into WebAssembly format for use in web pages. This allows you to write high-performance applications that run directly in the browser.
```

Comment:
WebAssembly is the binary instruction format, so we shouldn't say that "WebAssembly code can be written in a variety of programming lanaguages"  Instead, I would say something like, "Code written in a variety of programming languages, including C, C++, Rust and others, can be compiled into WebAssembly format..."